cypress/pom/accountPage.ts

```diff
@@ -0,0 +1,34 @@
+import { should } from "chai";
+import { BasePage } from "./base/basePage";
+
+export class AccountPage extends BasePage {
+  constructor() {
+    super('#my-account', 'AccountPage', 'index.php?controller=my-account')
+  }
+
+  private get userInfo(): Cypress.Chainable {
+    return this.container.find('a[title = "View my customer account"]', { timeout: 10000 })
+  }
+
+  private get bodyText(): Cypress.Chainable {
+    return this.container.find('#center_column', { timeout: 10000 })
+  }
+
+  public checkUserInfo(): this {
+    cy.allure().startStep(`Check user Info in header`)
+    this.userInfo
+      .contains('John Wick');
```

Comment:
In your case, you need to parametrize your method.Because username might have another value, for example 'Bill Gates'; 
So you need to rewrite your method using like:
`.checkUserInfo('John Wick');`